### PR TITLE
Adding Cypress to tfgrid_dashboard

### PR DESCRIPTION
**Pull request by mohamedamer453**
### Description

- Adding the first Cypress example to tfgrid_dashboard.
- Linking the added example with a github workflow.

### Related Issues

#163

### CODE REVIEW

File: cypress/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+class Utils{
+
+    formatBytes(bytes, decimals = 2) {    
+        const k = 1000;
```
Comment (AhmedHanafy725) [inline cypress/utils/utils.ts]:
```suggestion
        const k = 1024;
```

File: cypress/e2e/verify_explorer_stats.cy.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+import statsPage from "../page-objects/stats-page"
+import utils from "../utils/utils"
+
+
+describe('TF Grid Dashboard', function(){
+
+
+    this.beforeEach(function(){
+        cy.visit("/explorer/statistics")
```
Comment (AhmedHanafy725) [inline cypress/e2e/verify_explorer_stats.cy.ts]: let's put the visit request on the stats page model

File: package.json [hunk 3]
```diff
@@ -9,14 +9,14 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "1x": "^0.0.0",
```
Comment (AhmedHanafy725) [inline package.json:12]: why removing this dependency

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
- rename the `Docs` dir to `docs`
- please use yarn in installation, and add the changes to the yarn.lock file

File: package.json (hunk 3 above)
Comment (mohamedamer453) [reply package.json:12]: it was not removed but for some reason it was moved down a couple of lines to to `line 19`

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**